### PR TITLE
Added program sequence service to get program sequence id for a user

### PR DESCRIPTION
**Pull request by Satendra-SR**
#323 

### Types of changes

- [ ] Bug fix (a change which fixes an issue)
- [x] New feature (a change which adds functionality)

### Short description of what this resolves:

Why these change required? What problem does it solve?
- Added program sequence service to get program sequence id for a user

### Checklist:

- [x] I have performed a self-review of my own code.
- [x] The code follows the style guidelines of this project.
- [x] The code changes are passing the CI checks
- [ ] I have documented my code wherever required.
- [ ] The changes requires a change to the documentation.
- [ ] I have updated the documentation based on the my changes.

### CODE REVIEW

File: api/services/program_sequence_service.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            )
+
+            if not user_program_details:
+                user_program_details = models.UserProgram.query.get_latest_user_program(
+                    user_details.id
+                )
+
+            if not user_program_details:
+                # Need to log this
+                return None, None
```
Comment (GauravGusain98) [inline api/services/program_sequence_service.py:46]: Why returning 2 parameter and you are expecting only one.

File: api/services/program_sequence_service.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        self.user_phone = helpers.sanitize_phone_string(user_phone)
+
+    def get_program_sequence_id(self, jsonData):
+        self.set_init_data(jsonData)
+        try:
+            user_details = models.User.query.get_by_phone(self.user_phone)
+            if not user_details:
+                # Need to log this
+                print("User not available")
+                return None, None
```
Comment (GauravGusain98) [inline api/services/program_sequence_service.py:21]: Why returning 2 parameter and you are expecting only one.

File: api/services/program_sequence_service.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+            module_content_details = models.ModuleContent.query.get_by_content_id(
+                self.content_id
+            )
+
+            if not module_content_details:
+                # it is optional to have Content and Module association
+                return None, None
```
Comment (GauravGusain98) [inline api/services/program_sequence_service.py:29]: Why returning 2 parameter and you are expecting only one.

**Review by GauravGusain98 — APPROVED** [review]
LGTM

File: api/services/program_sequence_service.py (hunk 1 above)
Comment (Satendra-SR) [reply api/services/program_sequence_service.py]: Updated

File: api/services/program_sequence_service.py (hunk 3 above)
Comment (Satendra-SR) [reply api/services/program_sequence_service.py]: Updated

**Pull request merged**